docs/updates/2020-08-07-announce-book.md

```diff
@@ -0,0 +1,108 @@
+# 2020-08-07 - Announcing the new Jupyter Book
+
+*Note: this announcement is cross-posted between the [Jupyter Blog](https://blog.jupyter.org) and the [Executable Book Project updates blog](https://executablebooks.org/en/latest/updates/index.html)*
+
+Jupyter Book is an open source project for building beautiful, publication-quality books and documents from computational material. With this post, we're happy to announce that Jupyter Book has been re-written from the ground up, making it easier to install, faster to use, and able to create more complex publishing content in your books. It is now supported by [the Executable Book Project](https://executablebooks.org/en/latest/), an open community that builds open source tools interactive and executable documents in the Jupyter ecosystem and beyond.
+
+```{figure} images/jb-ui.png
+
+The new Jupyter Book interface, highlighting how you can [insert code outputs into your content](https://jupyterbook.org/content/glue.html)
+```
+
+This post is a short overview of the new features in Jupyter Book, and gives some background on the future of the project.
+
+You can also browse the [new Jupyter Book documentation](https://jupyterbook.org), or [check out the Jupyter Book GitHub repository](https://github.com/executablebooks/jupyter-book).
+
+**Note**: Jupyter Book is still in beta, so things are always changing. We'd love your feedback, ideas, and PRs for how to make the project better!
+
+## What does the new Jupyter Book do?
+
+The new version of Jupyter Book will feel very similar. However, it has a lot of new features due to the new Jupyter Book stack underneath (more on that later).
+
+The new Jupyter Book has the following main features (with links to the relevant documentation for each):
+
+* **[Write publication-quality content in markdown](https://jupyterbook.org/content-types/markdown)**. You can write in either Jupyter markdown, or in [MyST Markdown, an extended flavor of markdown for scientific publishing](https://jupyterbook.org/content/myst).  This includes support for rich syntax such as [citations and cross-references](https://jupyterbook.org/content/citations), [math and equations](https://jupyterbook.org/content/math), and [figures](https://jupyterbook.org/content/figures).
```

Comment:
nitpick, but I don't feel that the main features (i.e. the text in bold) stands out very well from the rest of the text, especially due to all the other links, it all just kind of blurs into one. Perhaps have the subsequent text as sub-bullet points or something?

<img width="599" alt="image" src="https://user-images.githubusercontent.com/2997570/89692610-1abe5800-d904-11ea-999a-78f587312835.png">

Reply:
@chrisjsewell what do you think about the layout on the current jupyterbook.org home page? Is that better?

Reply:
https://github.com/executablebooks/jupyter-book/pull/864

Reply:
I've updated this so that it mimics the jupyterbook.org front page